oe_theme.info.yml

```diff
@@ -47,3 +47,7 @@ config_devel:
     - core.entity_view_display.media.remote_video.oe_theme_main_content
     - core.entity_view_display.media.video_iframe.oe_theme_main_content
     - core.entity_view_mode.media.oe_theme_main_content
+
+'interface translation project': oe_theme
```

Comment:
This should go in the helper module, not theme. You implemented `hook_locale_translation_projects_alter ` i nthe module..

Reply:
moved to oe_theme_helper (including translation file)